packages/plugin-core/src/commands/Sync.ts

```diff
@@ -48,14 +48,27 @@ export class SyncCommand extends BasicCommand<CommandOpts, CommandReturns> {
         severity: ERROR_SEVERITY.FATAL,
       });
 
-    const committed = await workspaceService.commitAndAddAll();
-    L.info(committed);
+    const { committed, pulled, pushed } = await window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+        title: "Syncing Workspace",
+        cancellable: false,
+      },
+      async (progress) => {
+        progress.report({ message: "committing repos" });
+        const committed = await workspaceService.commitAndAddAll();
```

Comment:
nit: can you also call `progress.increment()` - I think this gives a nice progress bar progression for users to see.  Maybe 25% for committed, 50% for pull, final 25% for push?

Reply:
a66b2de